crates/.gitignore

```diff
@@ -0,0 +1,10 @@
+# Generated by Cargo
```

Comment:
Generally I'm not fond a of a `crates` directory, seen that in some repos that look to be "only" rust.
However since the primary purpose of this repo is to house the proto files and as a convenience we're providing rust crates it seemed reasonable

Reply:
Did the interleaving, I verified buf.build only uploads protos and not other dirs/files.
The generated files will be `mobilecoin.attestation.v1.rs` so I think that should work